pallets/creditcoin/src/lib.rs

```diff
@@ -236,6 +237,10 @@ pub mod pallet {
 	#[pallet::getter(fn gate_contract)]
 	pub type GATEConract<T: Config> = StorageValue<_, DeployedContract, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn gate_faucet_address)]
+	pub type GATEFaucetAddress<T: Config> = StorageValue<_, T::AccountId, OptionQuery>;
```

Comment:
I kinda prefer `Account` instead of `Address`, since the term `address` feels sort of overloaded (register address, external address)

Reply:
Will create a follow up PR to rename `GATEFaucetAddress` to `GATEFaucetAccount`  and remove the `#transacational` attribute

Reply:
#1284